autosubmit/notifications/mail_notifier.py

```diff
@@ -1,23 +1,22 @@
-#!/usr/bin/env python3
-
-# Copyright 2015-2020 Earth Sciences Department, BSC-CNS
-
-# This file is part of Autosubmit.
-
-# Autosubmit is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-
-# Autosubmit is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-
-# You should have received a copy of the GNU General Public License
-# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
+ # Copyright 2015-2025 Earth Sciences Department, BSC-CNS 
+ # 
+ # This file is part of Autosubmit. 
+ # 
+ # Autosubmit is free software: you can redistribute it and/or modify 
+ # it under the terms of the GNU General Public License as published by 
+ # the Free Software Foundation, either version 3 of the License, or 
+ # (at your option) any later version. 
+ # 
+ # Autosubmit is distributed in the hope that it will be useful, 
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ # GNU General Public License for more details. 
+ # 
+ # You should have received a copy of the GNU General Public License 
+ # along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Every line of the header has a space before the `#` sign.

![image](https://github.com/user-attachments/assets/ebd70d63-79ec-48cb-b182-b98ad9218246)